query_execution/WorkOrdersContainer.hpp

```diff
@@ -311,16 +311,17 @@ class WorkOrdersContainer {
     }
 
     inline void addWorkOrder(WorkOrder *workorder) {
-      workorders_.push(workorder);
+      workorders_.emplace(workorder);
```

Comment:
Are you sure that this after this line, there will exist only a single pointer to ``workorder``? What happens to the
original pointer ``workorder``? For your reference, check out the discussion on this link - http://stackoverflow.com/questions/26094274/cannot-understand-how-to-add-new-object-to-stdliststdunique-ptrclassname